src/adr

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -e
-eval "$($(dirname $0)/adr-config)"
+eval "$("$(dirname "$(readlink -f "$0")" )"/adr-config)"
```

Comment:
Unfortunately `readlink -f`doesn't work on MacOS. Since this script is already bash-specific, this should work:
```
eval "$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null && pwd)/adr-config"
```